README.md

```diff
@@ -32,15 +32,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? Semantic HTML describes a web page. It doesn't modify the page itself in any way, but it allows search engines and developers reading the file to get an understanding of what the page it
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+2. Name two big differences between ```display: block;``` and ```display: inline;```. Block code bullies itself onto it's own line and will push other elements to the next row. Inline will remain on the same row as other inline elements. Inline elements will also ignore any height and width properties added to it, whereas block elements can be modified in this way. 
 
-3. What are the 4 areas of the box model?
+3. What are the 4 areas of the box model? Margin, padding, border, content
 
-4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+4. While using flexbox, what axis does the following property work on: ```align-items: center```? main-axis of the container
```

Comment:
The correct answer is the cross axis where x and y meet. This is the true center and it aligns vertical ly everything along that center point so items varieting in size will be center the same vertically